static/src/js/containers/SavedProjectsContainer/__tests__/SavedProjectsContainer.test.jsx

```diff
@@ -178,18 +179,27 @@ describe('SavedProjectsContainer', () => {
       const { props } = setup()
 
       const removeBtn = await screen.findByRole('button', { name: /remove project/i })
-      await userEvent.click(removeBtn)
-
-      expect(props.dispatchHandleError).toHaveBeenCalledTimes(1)
-      expect(props.dispatchHandleError).toHaveBeenCalledWith(
-        expect.objectContaining({
-          error: expect.any(Error),
-          action: 'handleDeleteSavedProject',
-          resource: 'project',
-          verb: 'deleting',
-          notificationType: 'banner'
-        })
-      )
+
+      await act(async () => {
+        const { user } = setup()
```

Comment:
Why is setup called here? It was already called on line 179, you should only be rending the component once.